src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/Presentation/QuickInfoPresenter.QuickInfoPresenterSession.cs

```diff
@@ -100,6 +100,12 @@ private void OnEditorSessionDismissed()
 
             internal void AugmentQuickInfoSession(IList<object> quickInfoContent, out ITrackingSpan applicableToSpan)
             {
+                if (_item == null)
```

Comment:
It still feels weird to me that we choose to create an item that doesn't do anything vs. just not having one in the first place. Put another way, why is this being done at the controller level vs. something that's just in the presentation level, since this behavior isn't something we'd necessarily want in other editors...?